app/Http/Controllers/CompetitionsController.php

```diff
@@ -39,7 +48,9 @@ public function show(Competition $competition)
     {
         $contest = Contest::find($competition->contest_id);
 
-        return view('competitions.show', compact('competition', 'contest'));
+        $users = $this->repository->getAll($competition->users->pluck('id')->toArray());
```

Comment:
I might end up changing [this](https://github.com/DoSomething/gladiator/blob/035239d1ca206f76297ac41a6924e57f98a4e7d9/app/Repositories/DatabaseUserRepository.php#L92) to use `->all()` but not really a huge deal. Just less processing if expected `items` property is always contains an array anyhoo.